core/controllers/reader.py

```diff
@@ -291,7 +351,13 @@ class StateHitEventHandler(base.BaseHandler):
 
     @require_playable
     def post(self, exploration_id):
-        """Handles POST requests."""
+        """Handles POST requests.
+        Records that a learner has hit new state in the exploration.
```

Comment:
new state --> a new state

Move this sentence to the end of the previous line.